controller/src/controller/exceptions.py

```diff
@@ -281,6 +282,10 @@ def to_sentry(self) -> SentryEventInfo:
         return info
 
 
+class DuplicateUrlError(SlackWebException):
```

Comment:
Not a web exception (it doesn't originate from an httpx exception).

Reply:
I picked that one because "url" is a field, and that's the only relevant information, but I guess I can just put it in the SlackException message.

Reply:
Done.